src/components/roles/RoleCard.tsx

```
@@ -0,0 +1,41 @@
+  return (
+    <div className="role-card bg-white rounded-lg shadow p-5 border-l-4 border-amber-700 hover:shadow-lg transition">
+      <div className="flex justify-between items-start mb-3">
+        <div>
+          <h3 className="text-lg font-semibold">{role.name}</h3>
+          <span className={`text-sm ${role.active ? 'text-green-600' : 'text-gray-500'}`}>
+            {role.active ? 'Active' : 'Inactive'}
+          </span>
+        </div>
+        <div className="flex space-x-2">
+          <button onClick={onEdit} className="text-blue-500 hover:text-blue-700">
+            <Edit3 className="w-5 h-5" />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)